src/helpers/isAllFieldsFilled.ts

```diff
@@ -0,0 +1,12 @@
+import { Course } from '../interfaces/index';
+
+export function isAllFieldesFilled(course: Course) {
+	if (
+		Object.values(course).some(
+			(value) => !value || (Array.isArray(value) && value.length === 0)
+		)
+	) {
+		alert('Fill all the fieldes');
+		throw new Error('not all fieldes were filled');
+	}
```

Comment:
```suggestion
	        return !Object.values(course).some(
			(value) => !value || (Array.isArray(value) && value.length === 0)
		)
```
this is an approximate example

Reply:
done